build.gradle.kts

```diff
@@ -786,7 +789,10 @@ tasks {
       ":catalogs:catalog-jdbc-mysql:copyLibAndConfig",
       ":catalogs:catalog-jdbc-postgresql:copyLibAndConfig",
       ":catalogs:catalog-hadoop:copyLibAndConfig",
-      ":catalogs:catalog-kafka:copyLibAndConfig"
+      ":catalogs:catalog-kafka:copyLibAndConfig",
+      ":bundles:aliyun-bundle:copyLibAndConfig",
+      ":bundles:aws-bundle:copyLibAndConfig",
+      ":bundles:gcp-bundle:copyLibAndConfig"
```

Comment:
Why do we need this? We will not ship these jars with gravitino binary, if you want to use it for test, you'd better figure out a different way.

Reply:
OK, let me try another way to copy the bundle jars when testing S3 in deploy mode automatically.

Reply:
Removed, I removed it and replace it with adding test dependency in `hadoop-catalog` and `hadoop3-filesystem`.